pycytominer/operations/get_na_columns.py

```diff
@@ -33,7 +33,7 @@ def get_na_columns(population_df, features="infer", samples="all", cutoff=0.05):
     """
 
     if samples != "all":
-        population_df.query(samples, inplace=True)
+        population_df = population_df.query(samples)
```

Comment:
Similar comment here: consider adding more detail through a code comment which describes what's happening here and why.

Reply:
I have fixed this problem, but I've also noticed that there wasn't a check for the cutoff value. Therefore, I added a check and some tests to reflect this implementation.